lld/test/COFF/arm64x-export.test

```diff
@@ -74,6 +74,10 @@ IMPLIB-EC-NEXT: Symbol: func
 IMPLIB-EC-NEXT: Symbol: __imp_aux_func
 IMPLIB-EC-NEXT: Symbol: #func
 
+RUN: FileCheck --check-prefix=OUT-DEF %s < out.def
+OUT-DEF:      EXPORTS
+OUT-DEF-NEXT:     func @1
```

Comment:
Is this enough to actually disambiguate between the two function namespaces here? Wouldn't this test pass even if we'd export the native symbol table instead?

Reply:
I added a test to the linker invocation that exports only the EC symbol (verifying that the command-line option applies only to EC symbols). I also confirmed that the test fails if I revert the change.

Reply:
Ah, right - this is a case where we only export on the EC side, due to the `-export:` option only applying there. Not entirely obvious, but as long as it technically covers this change, I guess it's fine.

(For practical use, `-output-def` is more commonly used when exports aren't explicitly specified but just exporting all symbols - but for testing I guess this is fine.)